napari/layers/_shapes_layer/shape_util.py

```diff
@@ -255,6 +255,29 @@ def orientation(p, q, r):
     return val
 
 
+def is_colinear(points):
+    """Determines is a list of 2D points are colinear
+
+    Parameters
+    -------
+    points : np.ndarray
```

Comment:
```suggestion
    points : (N, 2) array
```

Reply:
fixed - but will need to change in other places, worth doing in a follow-up refactor PR